lib/commonio.c

```diff
@@ -1,33 +1,10 @@
 /*
- * Copyright (c) 1990 - 1994, Julianne Frances Haugh
- * Copyright (c) 1996 - 2001, Marek Michałkiewicz
- * Copyright (c) 2001 - 2006, Tomasz Kłoczko
- * Copyright (c) 2007 - 2011, Nicolas François
- * All rights reserved.
+ * SPDX-FileCopyrightText: 1990 - 1994, Julianne Frances Haugh
+ * SPDX-FileCopyrightText: 1996 - 2001, Marek Michałkiewicz
+ * SPDX-FileCopyrightText: 2001 - 2006, Tomasz Kłoczko
+ * SPDX-FileCopyrightText: 2007 - 2011, Nicolas François
```

Comment:
How about removing spaces from the year ranges?

Reply:
Hmm, from the rest of the files this seems like a conscious style decision, not an oversight.

Reply:
Yes, I'm maintaining the spacing from the original copyright statements.  Not sure how that fits with spdx specs, but it makes it easier to read IMO.